### PR TITLE
[14.0][FIX]date_range: store date_end in date.range.generator

### PR DESCRIPTION
**Pull request by abenzbiria**
Fix for issue  : https://github.com/OCA/server-ux/issues/821

### CODE REVIEW

**Comment by abenzbiria** [comment]
Hi @lmignon ,

Can you please check this issue fix.


Thank you

**Comment by abenzbiria** [comment]
Hi @Dranyel-Bosd , rousseldenis , @CasVissers-360ERP
Thank you for your review and comments

File: date_range/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Date Range",
     "summary": "Manage all kind of date range",
-    "version": "14.0.2.1.3",
+    "version": "14.0.2.1.4",
```
Comment (lmignon) [inline date_range/__manifest__.py]: @abenzbiria The version bump will be done at merge time. Can you remove this change PLZ?

Comment (abenzbiria) [reply date_range/__manifest__.py]: Thanks @lmignon , done.

**Review by lmignon — APPROVED** [review]

**Comment by abenzbiria** [comment]
Hi @bealdav , LoisRForgeFlow

Could you please have a look to this PR

Thanks for help

**Review by bealdav — APPROVED** [review]

**Pull request merged**